docker-compose.yml

```diff
@@ -363,6 +363,9 @@ services:
   transactions-consumer:
     <<: *sentry_defaults
     command: run consumer ingest-transactions --consumer-group ingest-consumer
+  ingest-feedback-events:
```

Comment:
nit: could you change this to `feedback-events-consumer`? It matches the naming convention of the other ingest consumers

Reply:
Sure, should it be changed in create-kafka-topics.sh too? It's named ingest-feedback-events in prod so I was worried bout keeping consistent with that

Reply:
Nope, this is just the name of the service. We need the topic to stay as `ingest-feedback-events` since that is the topic being published/consumed from in prod

Reply:
Ok so only change line 366 to feedback-events-consumer?

Reply:
yep!